Calendar/Calendar/CalendarForm.cs

```diff
@@ -63,19 +75,29 @@ private void Form1_Load(object sender, EventArgs e)
 
         }
 
+        /// <summary>
+        /// Displays next month in calendar view. Sets currentDate to next month and updates the view
+        /// </summary>
         public void NextMonth()
         {
             _currentDate = _currentDate.AddMonths(+1);
             UpdateCalendarView();
-            DisplayCurrentMonth();
+            UpdateCurrentMonthText();
         }
 
+        /// <summary>
+        /// Displays previous month in calendar view. Sets currentDate to previous month and updates the view
+        /// </summary>
         public void PreviousMonth()
         {
             _currentDate = _currentDate.AddMonths(-1);
             UpdateCalendarView();
-            DisplayCurrentMonth();
+            UpdateCurrentMonthText();
         }
+
+        /// <summary>
+        /// Creates 42 controls for days of the month, 7 column and 6 rows. 
```

Comment:
Nie hardkodujemy informacji z kodu ile jest kontrolek wierszy i kolumn itp itd to może się zmienić i gwarantuje Ci, że nikt potem nie poprawi komentarza.
Wystarczy napisać, że inicjalizujemy kontrolki do wyświetlania widoku kalendarza i podpinamy eventy pod nie.